Resources/Prototypes/Traits/skills.yml

```diff
@@ -0,0 +1,14 @@
+- type: trait
+  id: Thieving
+  category: Uncategorized
```

Comment:
Nothing should be in `Uncategorized`.

Reply:
CPR training on SiN is on Uncategorized, would I create a Skills category or move it to Physical/Mental?

Reply:
I only had it in Uncategorized there as a placeholder because we really haven't looked into(or have had a reason to) redo the categories yet. You can move it to Physical for now I suppose, because the categories themselves are inherently a placeholder. We can't reorganize and remake the categories yet for traits because there simply aren't enough traits in the first place to justify doing it.

Reply:
SiN's not got our standards.
Probably put it in... uhhhh.. one of those two :)

Reply:
> CPR training on SiN is on Uncategorized, would I create a Skills category or move it to Physical/Mental?

Skills; when we get a Skills system in whichever form this can be moved elsewhere or something :shrug:

Reply:
> > CPR training on SiN is on Uncategorized, would I create a Skills category or move it to Physical/Mental?
> 
> Skills; when we get a Skills system in whichever form this can be moved elsewhere or something 🤷

Maybe.